### PR TITLE
fix to collection

### PR DESCRIPTION
**Pull request by vzhenia**

### CODE REVIEW

File: src/daemon/collector/content/collect.js [hunk 1]
```diff
@@ -48,13 +48,13 @@ function addToFrontAdQueue(ad) {
 }
 
 function getAdFromButton(qId,buttonId) {
-  // console.log('getAdFromButton(qId,buttonId)', qId,buttonId);
+  console.log('getAdFromButton(qId,buttonId)', qId,buttonId);
```
Comment (skinofstars) [inline src/daemon/collector/content/collect.js:51]: remove

File: src/daemon/collector/content/collect.js [hunk 2]
```diff
@@ -48,13 +48,13 @@ function addToFrontAdQueue(ad) {
...
   // console.log(frontadqueue);
   for (let i in frontadqueue) {
     if (frontadqueue[i].buttonId === buttonId) {
         let ad = frontadqueue[i];
         frontadqueue[i] = { raw_ad: "" };
-        // console.log('getAdFromButton(qId,buttonId) AD?', ad);
+        console.log('getAdFromButton(qId,buttonId) AD?', ad);
```
Comment (skinofstars) [inline src/daemon/collector/content/collect.js:57]: remove

File: src/daemon/collector/content/collect.js (hunk 1 above)
Comment (vzhenia) [reply src/daemon/collector/content/collect.js]: thanks. removed

File: src/daemon/collector/content/collect.js (hunk 2 above)
Comment (vzhenia) [reply src/daemon/collector/content/collect.js]: removed

**Pull request merged**